testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/security/add-security-domain.cli

```diff
@@ -0,0 +1,60 @@
+
+/subsystem=security/security-domain=idp:add
+
+/subsystem=security/security-domain=idp/authentication=classic:add( \
+    login-modules=[ \
+        { \
+            "code" => "UsersRoles", \
+            "flag" => "required", \
+            "module-options" => [ \
+                "usersProperties"=>"sts-users.properties", \
+                "rolesProperties"=>"sts-roles.properties" \
+            ] \
+        } \
+    ] \
+)
+
+/subsystem=security/security-domain=idp/authorization=classic:add( \
+    policy-modules=[ \
+        { \
+            "code" => "JACC", \
+            "flag" => "required" \
+        } \
+    ] \
+)
+
+/subsystem=security/security-domain=sp:add
+
+/subsystem=security/security-domain=sp/authentication=classic:add( \
+    login-modules=[ \
+        { \
+            "code" => "org.picketlink.identity.federation.bindings.jboss.auth.SAML2STSLoginModule", \
+            "flag" => "required", \
+            "module-options" => [ \
+                "configFile"=>"sts-config.properties", \
+                "password-stacking"=>"useFirstPass" \
+            ] \
+        }, \
+        { \
+            "code" => "UsersRoles", \
+            "flag" => "required", \
+            "module-options" => [ \
+                "usersProperties"=>"sp-users.properties", \
+                "rolesProperties"=>"sp-roles.properties", \
+                "password-stacking"=>"useFirstPass" \
+            ] \
+        } \
+    ] \
+)
+
+/subsystem=security/security-domain=sp/authorization=classic:add( \
+    policy-modules=[ \
+        { \
+            "code" => "JACC", \
+            "flag" => "required" \
+        } \
+    ] \
+)
+
+# set default EJB3 security domain (for some unknown reason is security domain annotation ignored as well as jboss-web.xml
+/subsystem=ejb3:write-attribute(name=default-security-domain,value=sp)
```

Comment:
The security domain annotation is not ignored, it is most likely you are using the incorrect one (you need org.jboss.ejb3.annotation.SecurityDomain), also EJB security domain is set via XML through jboss-ejb3.xml not jboss-web.xml